### PR TITLE
Update Type and names for release

### PR DESCRIPTION
**Pull request by jongio**

### CODE REVIEW

**Review by lmazuel — APPROVED** [review]

File: _data/releases/latest/dotnet-packages.csv [hunk 1]
```diff
@@ -15,7 +15,7 @@
 "Azure.Security.KeyVault.Certificates","4.1.0","4.2.0-beta.1","KeyVault - Certificates","Key Vault","keyvault","","","client","","Replaces: Microsoft.Azure.KeyVault"
 "Azure.Security.KeyVault.Keys","4.1.0","4.2.0-beta.1","KeyVault - Keys","Key Vault","keyvault","","","client","","Replaces: Microsoft.Azure.KeyVault"
 "Azure.Security.KeyVault.Secrets","4.1.0","4.2.0-beta.1","KeyVault - Secrets","Key Vault","keyvault","","","client","","Replaces: Microsoft.Azure.KeyVault"
-"Microsoft.Azure.Core.NewtonsoftJson","","1.0.0-preview.1","Microsoft.Azure.Core.NewtonsoftJson","","NA","","","client","",""
+"Microsoft.Azure.Core.NewtonsoftJson","","1.0.0-preview.1","Core NewtonsoftJson","","NA","","","client","",""
```
Comment (weshaggard) [inline _data/releases/latest/dotnet-packages.csv:18]: Should this be in the new libraries? It seems more like a utility then something folks would want to show up as a primary new library.

**Review by weshaggard — APPROVED** [review]

**Comment by weshaggard** [comment]
@jongio  looks like you need to fix the conflict with the java package list.

File: _data/releases/latest/dotnet-packages.csv (hunk 1 above)
Comment (jongio) [reply _data/releases/latest/dotnet-packages.csv:18]: I don't know.  @AlexGhiondea

Comment (jongio) [inline _data/releases/latest/dotnet-packages.csv:18]:
```suggestion
"Microsoft.Azure.Core.NewtonsoftJson","","1.0.0-preview.1","Core NewtonsoftJson","","NA","","","","",""
```

Comment (jongio) [reply _data/releases/latest/dotnet-packages.csv]: Alex wants it removed so I did.

**Pull request merged**